dvc/exceptions.py

```diff
@@ -3,8 +3,6 @@
 
 from funcy import first
 
-from dvc.utils import error_link, format_link, relpath
```

Comment:
`dvc.utils` imports a lot of things. Making it lazy here as `dvc.exceptions` is imported in almost all of the commands that will affect everything (even `dvc -v`)